apps/emqx/test/emqx_persistent_session_SUITE.erl

```diff
@@ -139,6 +139,7 @@ end_per_group(_, _Config) ->
     ok.
 
 init_per_testcase(TestCase, Config) ->
+    emqx_common_test_helpers:clear_screen(),
```

Comment:
I think it doesn't need to be all or nothing...  

If there is somewhere to place it that will affect things globally (I don't know where), it can be added there later, without affecting existing places (it does nothing the 2nd+ time it's called).  Even if this place doesn't exist, I don't think it's a reason to forbid it from being there, since it's a no-op if you don't use it.

Reply:
I don't see why persistent session suite should behave differently in terms of UI from all the other test suites. It should be a job of the test framework to check the user preferences and apply them consistently.

I think this call can be moved to `emqx_common_test_helpers:init_per_testcase`.

Reply:
Sorry, what do you mean.

As I mentioned before, it's a no-op when the `CLEAR_SCREEN=true` environment variable is not set...  So leaving it there won't make any difference at all, nothing jarring. 

And I can't see how that's related to UI.  :thinking:

Reply:
Can you explain to someone unfamiliar with the code, why does `emqx_persistent_session_SUITE` test suite cleans the display, while, for example, `emqx_flapping_SUITE` doesn't? Is it related to the business logic at hand?

Reply:
Both won't do such a thing, unless you have `CLEAR_SCREEN=true` set?  And I guess it's not a common environment variable to be set.  Do you think it may be commonly set by devs?